### PR TITLE
Enable nightlies for pg15

### PR DESCRIPTION
**Pull request by gurkanindibay**
Removed ol/7 since pg 15 could not be downloaded for ol/7. I'm working another PR to be able to exclude pg versions that is not being supported by os release pair. With this PR  ol/7 nightlies could be baked for pg versions other than pg15
Note: With the approval of @metdos I removed both pgaf community and enterprise nightlies. Sİnce citus enterprise is not being used I removed it as well

### CODE REVIEW

File: .github/workflows/build-citus-community-nightlies.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: Build and publish citus community nightly packages
 
 env:
-  MAIN_BRANCH: "all-citus"
+  MAIN_BRANCH: "all-citus-pg15-beta"
```
Comment (hanefi) [inline .github/workflows/build-citus-community-nightlies.yml:4]: Do we intend to keep this change, or is it temporary?

File: .github/workflows/build-citus-community-nightlies.yml [hunk 2]
```diff
@@ -27,7 +27,8 @@ jobs:
         platform:
           - el/7
           - el/8
-          - ol/7
+          # temporarily removed from nightlies since pg15 beta is not being downloaded for ol/7
+          # - ol/7
```
Comment (hanefi) [inline .github/workflows/build-citus-community-nightlies.yml:31]: Do we not create packages for all PG versions? If only PG15 does not work for ol/7 is there a way to disable only this combination?

**Review by hanefi — CHANGES_REQUESTED** [review]
Now that we want to remove the workflow for enterprise, how about removing the word community from the single workflow we have left for Citus?

File: .github/workflows/build-citus-community-nightlies.yml (hunk 1 above)
Comment (gurkanindibay) [reply .github/workflows/build-citus-community-nightlies.yml:4]: This is temporary. I will not merge the PR for all-citus-pg15-beta either. (https://github.com/citusdata/packaging/pull/956)

File: .github/workflows/build-citus-community-nightlies.yml (hunk 2 above)
Comment (gurkanindibay) [reply .github/workflows/build-citus-community-nightlies.yml:31]: There is no way for it now. I opened the PR in tools to be able to handle it
https://github.com/citusdata/tools/pull/238

Comment (gurkanindibay) [reply .github/workflows/build-citus-community-nightlies.yml:31]: This PR is to enable nightlies without waiting the merge od this enablement PR. After merge of this PR there will be no need to disable ol/7 . We will just exclude pg15

**Comment by gurkanindibay** [comment]
> Now that we want to remove the workflow for enterprise, how about removing the word community from the single workflow we have left for Citus?

I think we need more time for that. I've just removed community. Marlin still uses citus enterprise < 11. There may be some backports in the future for citus enterprise so it is too soon to make this changes

**Review by hanefi — APPROVED** [review]
Dont forget to revert back to all-citus branch before merging

**Comment by gurkanindibay** [comment]
> Dont forget to revert back to all-citus branch before merging

No need to. We need pg 15 nightlies so all-citus-pg15-beta. I will revert back after citus will be ready without '--without-pg-version-check' flag

**Pull request merged**